ChanHoLee275/행렬-테두리-회전하기.js

```diff
@@ -0,0 +1,32 @@
+function solution(rows, columns, queries) {
+    var answer = [];
+    const matrix = [... new Array(rows)].map((_, idx1) => 
+        [... new Array(columns)].map((_, idx2) => idx2 + 1 + idx1*columns) 
+    );
+    queries.forEach(el => {
+        const [row1, column1, row2, column2] = el;
```

Comment:
여기는 이렇게 쓰면 더 간단하게 쓸수 있겠군요!

```suggestion
    queries.forEach(([row1, column1, row2, column2]) => {
```